loleaflet/src/layer/tile/ScrollSection.ts

```diff
@@ -0,0 +1,345 @@
+/* eslint-disable */
+/* See CanvasSectionContainer.ts for explanations. */
+
+declare var L: any;
+declare var $: any;
+declare var Hammer: any;
+
+class ScrollSection {
```

Comment:
@gokaysatir: I'm missing a purpose description comment ;-) - can you please add it?